app/main/autoupdater.js

```diff
@@ -52,35 +52,27 @@ function appUpdater() {
 
 function updateNotification() {
 	const options = {
-		host: 'api.github.com',
-		path: '/repos/zulip/zulip-electron/releases/latest',
+		url: 'https://api.github.com/repos/zulip/zulip-electron/releases/latest',
 		headers: {'User-Agent': 'request'}
 	};
 
-	https.get(options, res => {
-		let json = '';
-		res.on('data', body => {
-			json += body;
-		});
-		res.on('end', () => {
-			if (res.statusCode === 200) {
-				try {
-					const data = JSON.parse(json);
-					if (semver.gt(data.tag_name, app.getVersion())) {
-						dialog.showMessageBox({
-							message: 'A new update is available',
-							detail: 'Version ' + data.tag_name + ' is available.'
-						});
-					}
-				} catch (err) {
-					console.log(err);
-				}
-			} else {
-				console.log('Status:', res.statusCode);
+	request(options, (error, response, body) => {
+		if (error) {
+			console.log('Error:', error);
+			return;
+		}
+
+		if (response.statusCode === 200) {
```

Comment:
We should use `response.statusCode < 400` here to avoid some problem in future. Sorry i missed this before.